pkg/stanza/operator/input/windows/xml.go

```diff
@@ -31,6 +31,7 @@ type EventXML struct {
 	Security         *Security   `xml:"System>Security"`
 	Execution        *Execution  `xml:"System>Execution"`
 	EventData        EventData   `xml:"EventData"`
+	RemoteServer     string      `xml:"RemoteServer,omitempty"`
```

Comment:
Good point @djaglowski - the same information is already on the event XML returned by the event APIs, see https://learn.microsoft.com/en-us/windows/win32/wes/eventschema-computer-systempropertiestype-element

We don't need this field.

Reply:
I went ahead and moved the remote server info to an attribute in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34720, but if it's always the same value then I suppose we don't even need the attribute. Any idea how to assess if these might sometimes be different in a meaningful way?

Reply:
Per doc above, `Computer` records where the event _happened_. This sounds to me that it can be different than the remote computer that we are targeting. @JonathanWamsley can you help us by testing that?